bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/sequencefiles/EndToEndIT.java

```diff
@@ -51,47 +47,34 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
+/**
+ * Integration test for export/import table.
+ *
+ * <p>Mandatory parameters:
+ *
+ * <pre>
+ *  -Dgoogle.bigtable.project.id=[bigtable project]
+ *  -Dgoogle.bigtable.instance.id=[bigtable instance id]
+ *  -Dgoogle.dataflow.gcsPath=gs://[google storage path]
+ * </pre>
+ */
 @RunWith(JUnit4.class)
 public class EndToEndIT {
-  // Location of test data hosted on Google Cloud Storage, for on-cloud dataflow tests.
-  private static final String CLOUD_TEST_DATA_FOLDER = "cloud.test.data.folder";
-
   // Column family name used in all test bigtables.
   private static final String CF = "column_family";
 
-  // Full path of the Cloud Storage folder where dataflow jars are uploaded to.
-  private static final String GOOGLE_DATAFLOW_STAGING_LOCATION = "google.dataflow.stagingLocation";
+  private static final String projectId = getTestProperty(BigtableOptionsFactory.PROJECT_ID_KEY);
+  private static final String instanceId = getTestProperty(BigtableOptionsFactory.INSTANCE_ID_KEY);
+  private static final String gcsPath = getTestProperty("google.dataflow.gcsPath");
+  private static final String gcsWorkDir = gcsPath + "/" + UUID.randomUUID();
```

Comment:
similar to above

Reply:
`google.dataflow.gcsPath` is introduced by this fix, fallback is not required